tests/dtypes/test_money.py

```diff
@@ -0,0 +1,43 @@
+from pytest import FixtureRequest
+from spinta.core.config import RawConfig
+from spinta.testing.client import create_test_client
+from spinta.testing.data import listdata
+from spinta.testing.manifest import bootstrap_manifest
+
+
+def test_money(
+    rc: RawConfig,
+    postgresql: str,
+    request: FixtureRequest,
+):
+    context = bootstrap_manifest(rc, '''
+    d | r | b | m | property                   | type           | ref
+    backends/postgres/dtypes/money             |                |
+      |   |   | City                           |                |
+      |   |   |   | name                       | string         |
+      |   |   |   | amount                     | integer        | 
+    ''', backend=postgresql, request=request)
+
+    model: str = 'backends/postgres/dtypes/money/City'
+
+    app = create_test_client(context)
+    app.authmodel(model, [
+        'insert',
+        'getall',
+    ])
+
+    # Write data
+    resp = app.post(f'/{model}', json={
+        'name': "Vilnius",
+        'amount': 100
+    })
+    assert resp.status_code == 201
+
+    # Read data
+    resp = app.get(f'/{model}')
+    assert listdata(resp, full=True) == [
+        {
+            'name': "Vilnius",
+            'amount': 100
+        }
+    ]
```

Comment:
Add a test, where an incorrect currency is given, you should only check if currency is 3 letters long.

Reply:
Can you explain where in the test should be checked currency length.

Reply:
Currency length should be check not in test, but in [simple_data_check](https://github.com/atviriduomenys/spinta/blob/master/spinta/commands/__init__.py#L224).

Example:
https://github.com/atviriduomenys/spinta/blob/7fbc6e33464611cc5ed00afc72e09d3327f372b8/spinta/backends/fs/commands/validate.py#L12-L23

Reply:
Can you explain how this function work? Where the length should be checked?

Is this test is correct for currency validate?

@pytest.mark.parametrize('value', [
    'EUR',
    'EEUR',

])
def test_currency_value(rc: RawConfig, postgresql: str, value: str):
    context = bootstrap_manifest(rc, '''
    d | r | b | m | property                   | type           | ref
    backends/postgres/dtypes/money             |                |
      |   |   | City                           |                |
      |   |   |   | name                       | string         |
      |   |   |   | amount                     | money          | 
      |   |   |   | currency                   | string         | 
    ''', backend=postgresql)

    store: Store = context.get('store')
    manifest: Manifest = store.manifest

    backend = cast(PostgreSQL, store.backends['default'])

    model = manifest.models['backends/postgres/dtypes/money/City']

    load_data = {
        '_op': 'insert',
        'currency': value,
    }

    context.set('transaction', backend.transaction(write=True))

    data = dataitem_from_payload(context, model, load_data)
    data.given = commands.load(context, model, load_data)
    commands.simple_data_check(context, data, model, backend)